lightwood/mixers/nn/nn.py

```diff
@@ -1,48 +1,164 @@
-import torch.nn as nn
-import torch.nn.functional as F
-import torch.optim as optim
-from torch.utils.data import Dataset, DataLoader
-import torch
 import math
+import itertools as it
+
+import torch
+from torch.optim.optimizer import Optimizer
+
+
+
+class Ranger(Optimizer):
```

Comment:
moce this to another helper file and import it

Reply:
oh, yes... I kept it in nn.py for easier debugging and forgot to move it, will do